bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/api/CatalogQuery.scala

```diff
@@ -18,9 +18,17 @@ case class MagnitudeQueryFilter(mc: MagnitudeConstraints) extends QueryResultsFi
 }
 
 sealed abstract class CatalogName(val id: String, val displayName: String) extends Product with Serializable {
-  def supportedBands: List[MagnitudeBand] = Nil
+
+  def supportedBands: List[MagnitudeBand] =
+    Nil
+
   // Indicates what is the band used when a generic R band is required
-  def rBand: MagnitudeBand = MagnitudeBand.UC
+  def rBand: MagnitudeBand =
+    MagnitudeBand.UC
+
+  def voTableVersion: VersionToken =
+    VersionToken.unsafeFromIntegers(1, 2)
```

Comment:
The associated VO Table version.  This is actually included in the query results and we should take it from there instead at some point.  In the meantime I needed a way to refer to the 1.3 schema for Gaia.